### DISCUSSION TITLE
I have a RTMP stream that works in VLC / homebridge but will not work in Frigate

### DISCUSSION THREAD

**Discussion by peterwalkowiak** [blakeblackshear/frigate#3488] [Ask A Question]
I have tried it in Frigate as the address (rtmp://IP:1935/local/somecharacters) and with -I in front of it like it is in home bridge. The address works in VLC and in home bridge without an issue.

Here is my code for  frigate:

`  nanit:
    ffmpeg:
      inputs:
        - path: rtmp://IP:1935/local/somecharacters
          roles:
            - detect
    detect:
      width: 1920
      height: 1080
      fps: 5
    objects:
      track:
        - person
        - dog
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: false
      retain:
        default: 3
    record:
      enabled: False
      retain:
        days: 0
        mode: active_objects
      events:
        retain:
          default: 4
          mode: active_objects
        pre_capture: 5
        post_capture: 15`

**Comment by NickM-27** [comment]
Please see the docs and the recommended input args for rtmp cameras: https://docs.frigate.video/configuration/camera_specific#rtmp-cameras
